python/ray/util/sgd/torch/training_operator.py

```diff
@@ -348,7 +362,7 @@ def train_epoch(self, iterator, info):
         self._train_loader = train_loader
         self._validation_loader = validation_loader
 
-        if self._wrap_distributed_sampler:
+        if self._use_ddp:
             logging.debug("Wrapping data loaders with DistributedSampler.")
 
             def with_sampler(loader):
```

Comment:
hmm why is this code duplicated between the training_operator and distributed torch runner?

Reply:
and then one of them uses wrap_distributed_sampler and one uses _use_ddp?

Reply:
Ah yeah good catch. The duplicate code in distributed torch runner should be removed. It's not actually being called anywhere.